ReleaseProcessAutomation/Steps/SubSteps/ReleaseVersionAndMoveIssuesSubStep.cs

```diff
@@ -59,6 +61,60 @@ public void Execute (SemanticVersion currentVersion, SemanticVersion nextVersion
       _jiraVersionReleaser.ReleaseVersionAndSquashUnreleased(currentVersionID, nextVersionID);
     else
       _jiraVersionReleaser.ReleaseVersion(currentVersionID, false);
+
+    return !movePreReleaseIssues || MoveClosedIssuesFromPreviousVersion(currentVersion);
+  }
+
+  private bool MoveClosedIssuesFromPreviousVersion (SemanticVersion currentVersion)
+  {
+    var currentFullVersion = currentVersion.GetCurrentFullVersion().ToString();
+    try
+    {
+      var currentVersions = _jiraVersionCreator.FindAllVersionsStartingWithVersionNumber(currentFullVersion)
+                            ?? throw new InvalidOperationException($"Could not find version '{currentFullVersion}'.");
```

Comment:
This will never return `null`. Either we remove the `throw` or change it to an empty check